modules/swagger-annotations/src/main/java/com/wordnik/swagger/annotations/SwaggerConfig.java

```diff
@@ -0,0 +1,110 @@
+/**
+ *  Copyright 2015 SmartBear Software
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.wordnik.swagger.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation that configures definition level metadata. Still missing are the following:
+ * - Security Definitions
+ * - Security Requirements
+ * - Parameters
+ * - Responses
+ */
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+public @interface SwaggerConfig {
```

Comment:
We have a SwaggerConfig interface (https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-core/src/main/java/com/wordnik/swagger/config/SwaggerConfig.java) so it's probably better to rename. If this is an interface, I'd just use the full name SwaggerConfiguration.